src/components/Link/Link.js

```diff
@@ -20,8 +22,14 @@ const Link = props => {
 Link.displayName = 'Link';
 
 Link.propTypes = {
+    /** The spinner context (e.g. brand, primary, bad, good etc.) */
+    context: PropTypes.oneOf(['default', 'muted', ...CONTEXTS]),
```

Comment:
Let's have only 'neutral' for now, until we really need more contexts.

Also, we currently don't have a 'default' context. The component has a default styling of its own (no prop applied), which is not a 'context'. This is also how its implemented in other components and I'd like to stick with that.